src/usr/local/www/interfaces.php

```diff
@@ -2837,6 +2848,13 @@ function build_port_list() {
 	$pconfig['provider']
 ))->setHelp('This field can usually be left empty.');
 
+$section->addInput(new Form_Input(
+	'hostuniq',
+	'Host-Uniq',
+	'text',
+	$pconfig['hostuniq']
+))->setHelp('Required by some ISP, usually can be left empty.');
```

Comment:
change to "A unique host tag value for this PPPoE client. Leave blank unless a value is required by the service provider."

Reply:
Fixed,

now  it uses correct syntax:
`set pppoe service "hostuniq|provider"`
see https://reviews.freebsd.org/D9270#229774
I see the correct host-uniq value in packet capture

also found the old PR https://github.com/pfsense/pfsense/pull/4048 with this feature